src/features/machines/swapIntentMachine.ts

```diff
@@ -68,18 +67,23 @@ type Context = {
   intentHash: string | null
   error: null | {
     tag: "err"
-    value: {
-      reason:
-        | "ERR_USER_DIDNT_SIGN"
-        | "ERR_CANNOT_VERIFY_SIGNATURE"
-        | "ERR_SIGNED_DIFFERENT_ACCOUNT"
-        | "ERR_PUBKEY_EXCEPTION"
-        | "ERR_CANNOT_PUBLISH_INTENT"
-        | "ERR_QUOTE_EXPIRED_RETURN_IS_LOWER"
-        | WalletErrorCode
-        | PublicKeyVerifierErrorCodes
-      error: Error | null
-    }
+    value:
+      | {
+          reason:
+            | "ERR_USER_DIDNT_SIGN"
+            | "ERR_CANNOT_VERIFY_SIGNATURE"
+            | "ERR_SIGNED_DIFFERENT_ACCOUNT"
+            | "ERR_PUBKEY_EXCEPTION"
+            | "ERR_CANNOT_PUBLISH_INTENT"
+            | "ERR_QUOTE_EXPIRED_RETURN_IS_LOWER"
+            | WalletErrorCode
+            | PublicKeyVerifierErrorCodes
+          error: Error | null
+        }
+      | {
+          reason: "ERR_CANNOT_PUBLISH_INTENT"
```

Comment:
also need to remove `            | "ERR_CANNOT_PUBLISH_INTENT"`

Reply:
If I remove it from the value above there is a collision in swap form